### DISCUSSION TITLE
请问套了cloudflare之后 还是走服务器的流量吗？

### DISCUSSION THREAD

**Discussion by zhujitop** [AlistGo/alist#2279] [Q&A]
alist的网站套了cloudflare cdn，然后添加网盘账号开启了Web 代理。

这种情况是不是下载流量走的服务器流量？
而不是cloudflare的流量？

因为我看我服务器的流量暴涨，而cloudflare的流量并没有用多少。

**Comment by xhwsd** [comment]
- 本地代理  = 网盘平台 --> ALIST服务端 --> 下载用户
- 使用代理地址 = 网盘平台 --> 代理地址服务 --> 下载用户
- 302重定向 = 网盘平台 --> 下载用户

> 如果我没理解错，大致就这样的！

**Reply by zhujitop** [reply]
> * 本地代理  = 网盘平台 --> ALIST服务端 --> 下载用户
> * 使用代理地址 = 网盘平台 --> 代理地址服务 --> 下载用户
> * 302重定向 = 网盘平台 --> 下载用户
> 
> > 如果我没理解错，大致就这样的！

也就是如果要用cloudflare的流量，选择使用代理地址？

因为我现在用的302重定向，是没有走cloudflare的流量。
